src/data/weapons.py

```diff
@@ -204,4 +204,61 @@
         "type": " sword",
         "size": 9
     }
-}
\ No newline at end of file
+}
+
+Range_Weapon_Data = {
+	"heavy_bow": {
+		"mat_types": ["metal"],
+		"notable": "BOWlder",
+		"size_string": "long",
+        	"pierce_trauma": 0.8,
+        	"blunt_trauma": 0.7,
+        	"type": "bow",
+        	"size": 8
+	},
+	"light_bow": {
+		"mat_types": ["wood"],
+        	"notable": "robinhood",
+        	"size_string": "long",
+        	"pierce_trauma": 0.8,
+        	"blunt_trauma": 0.2,
+        	"type": "bow",
+        	"size": 4
+	},
+	"crossbow": {
+		"mat_types": ["metal"],
+        	"notable": "BOWlder",
+        	"size_string": "long",
+        	"pierce_trauma": 0.9,
+        	"blunt_trauma": 0.3,
+        	"type": "bow",
+        	"size": 3
+	},
+	"slingshot": {
+		"mat_types": ["wood"],
+        	"notable": "birds_eye",
+        	"size_string": "long",
+        	"pierce_trauma": 0.1,
+        	"blunt_trauma": 0.2,
+        	"type": "bow",
+        	"size": 1
+	},
+	"ninja_dart": {
```

Comment:
Can't have ninja dart sorry. 
The theme needs to be along the lines of [Narnia.](http://narnia.wikia.com/wiki/The_Chronicles_of_Narnia_Wiki)